src/datatypes/numeric.js

```diff
@@ -24,7 +24,14 @@ function generateFunctions(bufferReader,bufferWriter,size)
     };
   };
   var writer=function(value, buffer, offset) {
-    buffer[bufferWriter](value, offset);
+    try {
```

Comment:
Maybe it should be something else than a console.log though but I'm not sure what